sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -875,6 +908,26 @@ private void setDefaultTransmissionTarget(String transmissionTargetToken) {
         }
     }
 
+    /**
+     * Set trigger interval. The interval should be between 3 seconds and 1 day.
+     * Should be called before the service is started.
```

Comment:
We should explicitly document the unit of measurement. Parameter name implies that this is seconds, but parameter name is not javadocs.